frontend/src/components/pages/CompleteProfilePage.tsx

```diff
@@ -1,18 +1,56 @@
-import React from "react";
+import React, { useState } from "react";
+import { useMutation } from "@apollo/client";
 import { Box, Button, Flex } from "@chakra-ui/react";
 import Header from "../navigation/Header";
 import UserProfileForm from "../userProfile/UserProfileForm";
+import {
+  UPDATE_ME,
+  UpdateMeResponse,
+} from "../../APIClients/mutations/UserMutations";
 
 const CompleteProfilePage = () => {
-  const onSubmitClick = () => {
-    window.location.href = `#/`;
+  const [name, setFullName] = useState("");
+  const [email, setEmail] = useState("");
+  const [educationalQualification, setEducationalQualification] = useState("");
+  const [languageExperience, setLanguageExperience] = useState("");
+
```

Comment:
it's jarring to signup with a first name, last name, and email, and then be directed to an empty complete profile page where this info is missing and requires being reentered. Can you fill the name + email fields with info from the user context? ie
```
const [authenticatedUser, setAuthenticatedUser] =
    useState<AuthenticatedUser>(currentUser);
```